### PR TITLE
Mobile Touch Event

### PR DESCRIPTION
**Pull request by firestar300**
I added the JS Touch Event to use Imago.js on mobile. I tested it on iPhone 6S.

### CODE REVIEW

File: src/imago.js [hunk 1]
```diff
@@ -36,8 +41,8 @@
       e.preventDefault();
       e.stopPropagation();
 
-      if (e.which !== 1)
-        return;
+      // if (e.which !== 1)
+      //   return;
```
Comment (evandroeisinger) [inline src/imago.js]: this condition disable right mouse button, do you commented it with which reason?

File: src/imago.js [hunk 2]
```diff
@@ -112,17 +147,44 @@
...
+      if(e.type == 'touchend') {
+        // Touch Event
+        self.tmp.mouse = {
+          x: e.changedTouches[0].pageX,
+          y: e.changedTouches[0].pageY,
+        };
+
+        document.addEventListener('touchmove', crop);
+        document.addEventListener('touchend', stopCropping);
+
```
Comment (evandroeisinger) [inline src/imago.js:159]: did you test if works using touch moves? i tried on my mobile device and it doesn't work

**Comment by evandroeisinger** [comment]
Don't forget to check if the tests aren't failing after your improvements: https://travis-ci.org/evandroeisinger/imago.js/builds/110951858

**Comment by evandroeisinger** [comment]
thank you for your contribution :+1:

File: src/imago.js (hunk 1 above)
Comment (firestar300) [inline src/imago.js]: Touch event returns e.which = 0

File: src/imago.js (hunk 2 above)
Comment (firestar300) [inline src/imago.js:159]: What is your mobile OS ?

Comment (evandroeisinger) [inline src/imago.js:159]: iOS 9, dragging works but cropping isn't

Comment (firestar300) [inline src/imago.js:159]: It works for me. Tested on Safari iPad iOS9 and Chrome iPhone 6 iOS9

https://youtu.be/qKUMkuW-tm4

File: src/imago.js (hunk 1 above)
Comment (firestar300) [inline src/imago.js]: I added a condition in my last commit :

if(e.type == 'mousedown' && e.which !== 1)
   return;

File: src/imago.js (hunk 2 above)
Comment (evandroeisinger) [inline src/imago.js:159]: nice video :vhs: but did you try to crop the image using your fingers? maybe the crop handlers are two small for cropping, increase handlers size to see if works :santa:

**Comment by evandroeisinger** [comment]
@firestar300 don't forget to fix the tests :+1:

**Pull request closed without merging**